scalableminds/webknossos
# Restore rotation when popping a branch point or deleting a node in flight mode

This bug is a regression and the functionality used to work earlier - this PR restores it.
I've used this occasion to remove some unused code (the skipDimensions stuff was not of any relevance as far as I could see). I don't think we need to bump the API version, I checked the user scripts on our main instance and only one script was supplying this `skipDimensions` parameter (as `false`, so it wasn't really using it).
Setting the flycam rotation in `orthogonal` mode doesn't do any harm as it is simply ignored. It makes the code simpler imo and also the flycam then already has the correct rotation when switching to `flight` or `oblique` mode.

### Mailable description of changes:
 - The viewport rotation at which a node was created in the flight or oblique mode is now again restored when this node is set as active. This happens for example when deleting nodes (`shift + space`) or when jumping to a branchpoint (`j`).

### Steps to test:
- Trace in flight mode, set nodes with different viewport rotations and use `shift + space` to delete nodes, `j` to jump to branchpoints, `s` to center the active node, ... - the rotation at which the node was created should always be restored

### Issues:
- fixes #1977

------
